lib/xapi/track.rb

```diff
@@ -91,20 +92,28 @@ def dir
     end
 
     def config
-      @config ||= JSON.parse(File.read(File.join(dir, "config.json")))
+      @config ||= JSON.parse(File.read(config_filename))
     end
 
-    def doc(topic)
-      path = File.join(dir, "docs", topic.upcase)
-      f = Dir.glob("%s.*" % path).first
-      case f
-      when /\.md/
-        File.read(f)
-      when /\.org/
-        Orgmode::Parser.new(File.read(f)).to_markdown
+    def config_filename
+      File.join(dir, "config.json")
+    end
+
+    def document_contents(topic)
+      filename = document_fiename(topic)
+      case filename
+      when /\.md$/
+        File.read(filename)
+      when /\.org$/
+        Orgmode::Parser.new(File.read(filename)).to_markdown
       else
-        ""
+        ''
```

Comment:
I'm curious why you changed this. Is this your text editor switching things up?

I don't feel strongly about double/single quotes, but tend to just use double-quotes by default everywhere. Mostly because

- no benchmark I've ever seen suggests that the performance difference is a bottleneck (until that one time when it is, in which case I want to see the numbers)
- it's a pain to switch out single quotes for double quotes when I decide that I need to interpolate something. Here that's obviously not an issue, but it catches me off guard when people switch up quotation marks in a codebase.